packages/core/src/lib/select/searchable-select.svelte

```diff
@@ -404,8 +404,11 @@ const handleKeydown = createHandleKey({
                 name={option.icon}
               />
             {/if}
-            <div class="flex flex-col">
-              <p class="text-wrap text-sm">
+            <div
+              class="flex flex-col"
+              style="overflow-wrap: anywhere;"
```

Comment:
Tailwind hasn't merged an alternative to `word-break: break-word`.

See discussion: https://github.com/tailwindlabs/tailwindcss/discussions/12127
See PR: https://github.com/tailwindlabs/tailwindcss/pull/12128

Reply:
I'm cool with this, but can we do it as a tailwind arbitrary-inside-brackets class instead of an inline style?

Reply:
:+1: 53735e13